README.md

```diff
@@ -7,7 +7,8 @@ exercise how to make the portifolios
 
 - html and css
 - figma
-
+## live demo
+https://abel-desu.github.io/Portfolio-setup-and-mobile-version-skeleton/
```

Comment:
- _(optional)_ Great job on deploying your app. I only suggest adding the link to your readme file in this format `[My Portfolio](https://abel-desu.github.io/Portfolio-setup-and-mobile-version-skeleton/)`. It kinda looks more professional
(you can change the `My Portfolio` with anything you want)